extensions/shared/external-media/sources/pexels.js

```diff
@@ -57,9 +57,21 @@ function PexelsMedia( props ) {
 	// Load initial results for the random example query.
 	useEffect( getNextPage, [] );
 
+	const searchFormEl = useRef( null );
+	useEffect( () => {
+		const searchInputEl = searchFormEl.current.elements[ 0 ];
```

Comment:
This may be risky. Should we be more specific, like finding it by `input[type="search"]`?

Reply:
Made more robust in https://github.com/Automattic/jetpack/pull/15868/commits/78c1f2348048715ece0fc735750b46cd2329551d.